geopyspark/geotrellis/catalog.py

```diff
@@ -453,3 +458,94 @@ def write(uri,
                                     tiled_raster_layer.srdd,
                                     TimeUnit(time_unit).value,
                                     IndexingMethod(index_strategy).value)
+
+
+class Attributes(object):
```

Comment:
Will `Attributes` ever be seen or used by the user? If not, it might be good to add a `_` at the beginning so that people know it's a backend class.